_posts/2017/2017-05-09-how-to-successfully-run-a-tech-blog.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "How to successfully run a company tech blog"
+date: "2017-05-09 10:00:00"
+image: blog-header/social-like.jpg
+categories: tech-stories
+authors: ["Rose"]
+---
+
+Since we develop e-commerce solutions at [ePages](https://www.epages.com/us/), there were enough techies willing to help us out when we first decided to set up a [company tech blog](https://developer.epages.com/blog) in spring 2015.
+Our [API documentation](https://developer.epages.com/apps) was already [Jekyll](https://jekyllrb.com/)-based, so it came natural to connect our new blog to it.
+The new Social Media adventure, including the [blog](https://developer.epages.com/blog) as well as a [Twitter account](https://twitter.com/epagesdevs), started two years ago.
+
+In the following I would like to give you some deeper insights on what we have learnt ever since we began our blog journey in order to generate relevant content for our readers and how we engaged in the developer community via Twitter.
+
+## Where to start a new company tech blog?
+
+Once the technical framework was set, we were ready to go.
+But...where do you actually start when beginning to blog from zero?!
+
+Well, first of all we had to consider that a company's tech blog requires thoughtful planning.
+Planning your content ahead not only saves you lots of unnecessary stress when it comes to fresh content ideas, but also lets you see the big picture.
+Thus, it provides you with a great overview of how to integrate different topic areas into your content structure.
+For our ePages DevBlog we soon established the following topic categories:
+
+* Tech Stories
+* Working at ePages
+* Events
+* API
+* Agile
+
+With these categories in mind, we then developed an editorial calendar to determine what colleague would cover which topic area for which date.
+Usually, we have a rough content plan for the next two months and try to publish at least four times per months.
+We soon created a rotating system to make sure that our different developer teams take turns in writing blog posts.
+
+## Define guidelines for tech blog contributors
+
+Once you've determined the new blog's content structure, it is time to start with the creative writing process.
+For a company tech blog there will probably be more than only one contributor.
+Therefore, it is essential to define some basic guidelines for the blog post writers.
+At ePages, there are many developers, scrum masters and other colleagues from our R&D department that deliver great blog post ideas and content.
```

Comment:
You could link to the helpdocs, if you want: https://github.com/ePages-de/epages-docs/tree/develop/helpdocs